src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -56,6 +57,9 @@
                 'priority' => 100, // same priority than RequestAttributeValueResolver, but registered before
             ])
 
+        ->set('argument_resolver.datetime', DateTimeValueResolver::class)
+            ->tag('controller.argument_value_resolver', ['priority' => 150])
```

Comment:
I think we should put all new resolver at 105, including backed enum and uid resolver (then we can remove the comments that are found above)